src/org/infinity/datatype/Datatype.java

```diff
@@ -105,7 +104,7 @@ public int getSize()
   @Override
   public List<StructEntry> getStructChain()
   {
-    List<StructEntry> list = new Vector<StructEntry>();
+    final List<StructEntry> list = new ArrayList<>();
```

Comment:
Return value of this method not used from different threads so synchronized `Vector` not needed. Besides, in other places of interface implementation `ArrayList` is used